src/index.css

```diff
@@ -0,0 +1,55 @@
+body {
+  background-color: rgb(242, 231, 236);
+  display: flex;
+  justify-content: center;
+}
+
+.main-section {
+  margin-top: 10%;
+  display: flex;
+  flex-direction: column;
+  width: 300px;
+  height: max-content;
+  background-color: aquamarine;
+  border-radius: 25px;
+  padding: 25px;
+  gap: 2%;
+}
+
+.title {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.heading {
+  font-size: 30px;
+  font-weight: 400;
+}
+
+.bi-arrow-repeatt {
+  width: 40px;
+  height: 30px;
+  margin-top: 10%;
+  align-items: center;
+}
+
+.form-section {
+  display: flex;
+  justify-content: space-between;
+}
+
+.add {
+  width: 86%;
+  height: 30px;
+  border-radius: 5px;
+  border-color: antiquewhite;
+}
+
+.clear-btn,
+.bi-arrow-repeatt,
+.btn-add {
+  background-color: azure;
+  color: rgb(11, 243, 165);
+  border-radius: 7px;
+}
```

Comment:
| Given Design | Your Design |
| --- | --- |
| ![image](https://github.com/tajulafreen/To-Do-List/assets/93039359/14dc7af2-2beb-49d0-9a31-8cee92ec7f85) | ![image](https://github.com/tajulafreen/To-Do-List/assets/93039359/c94a9933-f259-4221-8f55-154a8dc2cd96) |

| Project Requirements 📝 |
| --- |
| ![image](https://github.com/tajulafreen/To-Do-List/assets/93039359/1f34a64a-d975-4058-934a-b1c03f26d203) |

- [ ] It would be nice if you follow the project requirements which clearly say that your project should be a clone of the project shown in the video so it would be better if you use the same layout, color scheme, etc 👍